src/twisted/newsfragments/7530.bugfix

```diff
@@ -0,0 +1 @@
+Made `ssl:` endpoints accept key.pem files without trailing newlines
```

Comment:
This actually affects the file specified by the `certKey`.  Could you change this to something like:

> The `ssl:` endpoint now accept `certKey` PEM files without trailing newlines.